zanata-war/src/main/java/org/zanata/action/VersionHomeAction.java

```diff
@@ -1092,6 +1093,7 @@ public void clearTransFileUpload() {
         translationFileUpload = new TranslationFileUploadHelper();
     }
 
+    // TODO turn this into a CDI bean?
```

Comment:
I think it can wait though. Don't you?

On 1 March 2016 at 09:55, Carlos Munoz <notifications@github.com> wrote:

> In zanata-war/src/main/java/org/zanata/action/VersionHomeAction.java
> <https://github.com/zanata/zanata-server/pull/1072#discussion_r54501182>:
>
> > @@ -1092,6 +1093,7 @@ public void clearTransFileUpload() {
> >          translationFileUpload = new TranslationFileUploadHelper();
> >      }
> >
> > +    // TODO turn this into a CDI bean?
>
> Yes, I think it can even be a dependent-scoped bean.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/1072/files#r54501182>.
>



-- 
Sean Flanigan

Principal Software Engineer
Globalisation Tools Engineering
Red Hat